package.json

```diff
@@ -16,7 +16,7 @@
   "scripts": {
     "build:clean": "rm -rf dist",
     "build": "npm run build:clean && rollup -c ./config/rollup.config.js",
-    "develop": "rollup -c ./config/rollup.config.js -w",
+    "dev": "npm run build -- -w",
```

Comment:
What does the `--` on its own do?

Reply:
it's used to pass in the `-w` param to the underlying script 

so effectively it runs `npm run build:clean && rollup -c ./config/rollup.config.js" -w`